banzai_nres/rv.py

```diff
@@ -91,9 +91,12 @@ class RVCalculator(Stage):
     MAX_ORDER_TO_CORRELATE = 101
 
     def do_stage(self, image) -> ObservationFrame:
+        if image.classification is None:
+            logger.warning('No classification to use for an RV template', image=image)
+            return image
```

Comment:
if the object is unclassified (not in gaia or simbad) then we will never do an RV measurement, do we want this?